impl/src/main/java/org/jboss/weld/logging/BeanLogger.java

```diff
@@ -522,4 +522,8 @@
     @Message(id = 1570, value = "Invalid BeanConfigurator setup - no callback was specified for {0}", format = Format.MESSAGE_FORMAT)
     IllegalStateException noCallbackSpecifiedForCustomBean(Object bean);
 
+    @LogMessage(level = Level.WARN)
+    @Message(id = 1571, value = "Proxy for {0} created in {1} because the class package is {2}.", format = Format.MESSAGE_FORMAT)
```

Comment:
Hm, I think it's the class which is signed so maybe this should be `Proxy for {0} created in {1} because {2}` and the reason should contain `the class...` etc.